lib/graphql/types.ts

```diff
@@ -319,26 +320,32 @@ export type Mutation = {
   createPermit: CreatePermitResult;
 };
 
+
```

Comment:
It would probably be nice to not add all of this extra whitespace (unless it was intentional?)

Reply:
These whitespaces were automatically generated by the script! I reran it so that any manual changes to `types.ts` get overwritten.